Makefile

```diff
@@ -84,27 +92,30 @@ Q = $(if $(filter 1,$V),,@)
 H = $(shell printf "\033[34;1m=>\033[0m")
 
 .PHONY: default
-default: depend build
+default: depend build test
 
 checkvars:
 	@if test -z "$(TAG)"; then echo "TAG missing"; exit 1; fi
 	@if test -z "$(HUB)"; then echo "HUB missing"; exit 1; fi
 
-setup: pilot/platform/kube/config
```

Comment:
Are all the dependencies on this file now removed?  If this target is removed then the "make setup" line in release/cloud_builder.sh should also be removed.

Reply:
Yes, they should be ( or if they're not the tests for this PR will catch it). We can clean it up in cloud builder as well, but let's see this passing...